### ISSUE TITLE
[1.21x] [Fabric] - Dev Env Data Gen Issue

### ISSUE THREAD

**Issue by xanthian** [KyaniteMods/DeeperAndDarker#345]
Trying to run datagen for my mod which utilises DeeperAndDarker assets

build.gradle.
```
    // Deeper and Darker 1.30-1.21
    modRuntimeOnly ("maven.modrinth:deeperdarker:LTl5Kmpc") { transitive false }
    modApi ("net.kyrptonaught:customportalapi:0.0.1-beta66-1.21")
    modApi ("io.wispforest:owo-lib:0.12.10+1.21")
```    
Error
```
[main/ERROR] (Minecraft) Failed to start the minecraft server
 java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'deeperdarker' at 'com.kyanite.deeperdarker.DeeperDarker'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:403) ~[fabric-loader-0.16.2.jar:?]
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.16.2.jar:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:401) ~[fabric-loader-0.16.2.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:63) ~[fabric-loader-0.16.2.jar:?]
	at net.minecraft.server.Main.main(Main.java:111) [minecraft-merged-d1cc4aff0f-1.21-net.fabricmc.yarn.1_21.1.21+build.9-v2.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) [fabric-loader-0.16.2.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.2.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.16.2.jar:?]
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) [dev-launch-injector-0.2.1+build.8.jar:?]
Caused by: java.lang.ExceptionInInitializerError
	at com.kyanite.deeperdarker.content.DDBlocks.<clinit>(DDBlocks.java:242) ~[deeperdarker-LTl5Kmpc.jar:?]
	at com.kyanite.deeperdarker.content.DDItems.<clinit>(DDItems.java:28) ~[deeperdarker-LTl5Kmpc.jar:?]
	at com.kyanite.deeperdarker.DeeperDarker.onInitialize(DeeperDarker.java:42) ~[deeperdarker-LTl5Kmpc.jar:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:399) ~[fabric-loader-0.16.2.jar:?]
	... 6 more
Caused by: java.lang.IllegalArgumentException: No enum constant net.minecraft.block.enums.NoteBlockInstrument.deeperdarker$SHATTERED
	at java.base/java.lang.Enum.valueOf(Enum.java:293) ~[?:?]
	at net.minecraft.block.enums.NoteBlockInstrument.valueOf(NoteBlockInstrument.java:8) ~[minecraft-merged-d1cc4aff0f-1.21-net.fabricmc.yarn.1_21.1.21+build.9-v2.jar:?]
	at com.kyanite.deeperdarker.content.blocks.ShatteredHeadBlock.<clinit>(ShatteredHeadBlock.java:24) ~[deeperdarker-LTl5Kmpc.jar:?]
	at com.kyanite.deeperdarker.content.DDBlocks.<clinit>(DDBlocks.java:242) ~[deeperdarker-LTl5Kmpc.jar:?]
	at com.kyanite.deeperdarker.content.DDItems.<clinit>(DDItems.java:28) ~[deeperdarker-LTl5Kmpc.jar:?]
	at com.kyanite.deeperdarker.DeeperDarker.onInitialize(DeeperDarker.java:42) ~[deeperdarker-LTl5Kmpc.jar:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:399) ~[fabric-loader-0.16.2.jar:?]

	
```
Error 2
```
Caused by: java.lang.IllegalArgumentException: No enum constant net.minecraft.block.enums.NoteBlockInstrument.deeperdarker$SHATTERED
```

**Comment by Pedro270707** [comment]
The bug was already fixed in 1.3.1

**Comment by xanthian** [comment]
so you did, apologies and thank you :)
